README.md

```diff
@@ -1,24 +1,61 @@
-# README
+# Every Kwacha App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Things you may want to cover:
+## Live Version
+Here is the [link](https://every-kwacha.herokuapp.com/) to the live version
 
-* Ruby version
+## Screenshots
 
-* System dependencies
+> | Splash Page | Login Page |
+> |--------------|----------------|
+> |![Screenshot](./images/Screenshot1.png)|![Screenshot](./images/Screenshot2.png)|!
+> | Home/Categories Page | Add Category Page |
+> |![Screenshot](./images/Screenshot3.png)|![Screenshot](./images/Screenshot4.png)|!
+> | Transactions Page | Add Transaction Page |
+> |![Screenshot](./images/Screenshot5.png)|![Screenshot](./images/Screenshot6.png)|!
 
-* Configuration
+## Built With
+- Ruby
+- Ruby on Rails
+- CSS
+- Bootstrap
+- RSpec
+- Capybara
 
-* Database creation
+## Setup
+- Get the link of the repository: [https://github.com/chaw-bot/every-kwacha](https://github.com/chaw-bot/every-kwacha)
+- Clone it as `git@github.com:chaw-bot/every-kwacha.git` on a Terminal
+- Usage
+- Run bundle install on a Terminal to get Gemfile dependencies.
+- Start the server by running `rails s`
 
```

Comment:
- [ ] I would recommend including all the steps to clone and run the project locally to make sure every user can use that easily. Currently, there are some steps missing here like how to create a database (`rails db:create`), migrate the database ( `rails db:migrate`).